policyengine_uk/variables/gov/hmrc/tax_free_childcare/conditions/tax_free_childcare_age_child_condition.py

```diff
@@ -0,0 +1,36 @@
+from policyengine_uk.model_api import *
+
+
+class tax_free_childcare_child_age_eligible(Variable):
+    value_type = bool
+    entity = Person
+    label = "Eligible child for the tax-free childcare"
+    documentation = "Whether this person meets the age and disability requirements for eligibility"
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        """
```

Comment:
Remove function docstrings for consistency with current codebase; label + documentation + value_type should cover it.